docs/tutorial/7-add-interactivity.md

```diff
@@ -71,12 +71,11 @@ async function startShell(terminal) {
       },
     })
   );
+  return shellProcess;
```

Comment:
This was inconsistent as I added it later but forgot it in certain places. This is also missing from the end product demo but is actually very important 😂  https://stackblitz.com/edit/stackblitz-webcontainer-api-starter-with-terminal?file=main.js. So could you make sure to add it there as well 🙏 ?

Reply:
Thank you.